rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -604,6 +609,7 @@ def get_history(
                             ),
                         ),
                         stake_operation=operation_stake,
+                        identifier=f"{change['transaction']['id']}-{change['transaction']['sequenceNumber']}",  # noqa: E501
```

Comment:
Why return an identifer that contains the transaction hash too and thus duplicates data? Just add the sequence number as a field instead and then the frontend will generate the composite key.

Reply:
as discussed in discord we keep it as is rn